logic/src/commonMain/kotlin/com/wire/kalium/logic/data/conversation/Conversation.kt

```diff
@@ -107,9 +107,9 @@ sealed class ConversationDetails(open val conversation: Conversation) {
             teamId = otherUser?.teamId,
             protocol = protocolInfo,
             mutedStatus = MutedConversationStatus.AllAllowed,
-            lastReadDate = null,
             lastNotificationDate = null,
             lastModifiedDate = lastModifiedDate,
+            lastReadDate = "1970-01-01T00:00:00.000Z",
```

Comment:
Can't we replace all these hardcoded string with something like `Instant.DISTANT_PAST.toString()`?